stream/src/test/java/com/annimon/stream/CollectorsTest.java

```diff
@@ -48,13 +54,74 @@ public void testToList() {
         assertThat(list, is(expected));
     }
 
+    @Test
+    public void testToUnmodifiableList() {
+        List<Integer> expected = Arrays.asList(0, 1, 2, 3, 4, 5);
+        List<Integer> list = Stream.range(0, 6)
+                .collect(Collectors.<Integer>toUnmodifiableList());
+        assertThat(list, is(expected));
+
+        try {
+            list.add(1);
+            fail("Expected an UnsupportedOperationException to be thrown when add item to list");
+        } catch (UnsupportedOperationException uoe) { }
```

Comment:
`expected` is used as name in Square projects for expected exceptions.
What about the same renaming for better readability?

Reply:
See https://github.com/square/okhttp/blob/878bde8fd6939b508adeb21e2911e390cfe27326/okhttp-tests/src/test/java/okhttp3/EventListenerTest.java#L165 for example